backstop/tests/breadcrumb.html

```diff
@@ -0,0 +1,163 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html id="theme">
+  <head>
+    <title>Breadcrumb | iTwinUI</title>
+    <link
+      rel="icon"
+      href="../assets/favicon.ico"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/all.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/demo.css"
+    />
+    <style>
+      .iui-input {
+        width: 450px;
+      }
+    </style>
+    <script src="../assets/theme.js"></script>
+  </head>
+  <body class="iui-body">
+    <theme-button></theme-button>
+    <h1>Breadcrumb</h1>
+    <hr />
+
+    <section id="demo-default">
+
+      <nav
+        class="iui-breadcrumb"
+        aria-label="Breadcrumb"
+      >
+        <button class="iui-button iui-borderless iui-dropdown">
+          <!-- Use folder.svg from @itwin/itwinui-icons package. -->
+          <svg
+            class="iui-icon"
+            aria-hidden="true"
+            viewBox="0 0 16 16"
+          >
+            <path d="M1,13a1,1,0,0,0,1,1H14a1,1,0,0,0,1-1V5a1,1,0,0,0-1-1H7V3A1,1,0,0,0,6,2H2A1,1,0,0,0,1,3Z" />
+          </svg>
+          <svg
+            class="iui-icon"
+            aria-hidden="true"
+            viewBox="0 0 16 16"
+          >
+            <path
+              d="m4.8067 6h6.3953a.27961.27961 0 0 1 .24043.44321l-3.1736 3.45707a.33969.33969 0 0 1 -.48085 0l-3.2217-3.45707a.26909.26909 0 0 1 .24042-.44321z"
+            />
+          </svg>
+        </button>
+        <ol>
+          <li
+            itemscope
+            itemtype="http://data-vocabulary.org/Breadcrumb"
+          >
+            <a
+              href="#"
+              itemprop="url"
+            >
+              <span itemprop="title">Root</span>
+            </a>
+          </li>
+          <li>
+            <span>&mldr;</span>
+          </li>
+          <li
+            itemscope
```

Comment:
Why these `item*` attributes are needed? What they give? Because I don't see them in W3 example.

Reply:
I got the `item*` attributes from [this article](https://coderwall.com/p/p0nvjw/creating-a-semantic-breadcrumb-using-html5-microdata).  From what I can tell, they help show the breadcrumb within search engines.

My wife does SEO and I asked her to look at this code.  She said 2 things:
1. She prefers to declare these in JSON, not inline HTML.
2. Link to schema.org instead of data-vocabulary.org.

Reply:
switching from data-vocabulary (which is deprecated btw) to schema.org requires more changes, and I think your syntax is wrong. See https://developers.google.com/search/blog/2020/01/data-vocabulary

But even then, I think this stuff should be left upto the user for two reasons:
- JSON-LD is really the recommended approach for structured data. If the user wanted to add that, it would conflict with this inline microdata.
- We don't want to have unnecessary SEO where not needed (e.g. breadcrumbs that don't change url)

Reply:
Ok, good reasons, I will remove all of the `item*` stuff.